docs/intro/overview.rst

```diff
@@ -189,68 +100,37 @@ You've seen how to extract and store items from a website using Scrapy, but
 this is just the surface. Scrapy provides a lot of powerful features for making
 scraping easy and efficient, such as:
 
-* Built-in support for :ref:`selecting and extracting <topics-selectors>` data
-  from HTML and XML sources
```

Comment:
I think XPath and CSS selectors should be mentioned explicitly, not only as a part of scrapy shell.